src/bentoml/_internal/configuration/__init__.py

```diff
@@ -113,6 +113,7 @@ def set_debug_mode(enabled: bool) -> None:
     )
 
 
+@lru_cache(maxsize=1)
```

Comment:
defintely should not cache this. 

should revert.

Reply:
ok

Reply:
If you just evict the last one, then what is the point of caching 😄 

Also it is just O(1) access, no need to cache.